apps/coscrad-frontend/src/components/resources/books/book-detail.full-view.presenter.tsx

```diff
@@ -10,26 +15,50 @@ export const BookDetailFullViewPresenter = ({
     author,
     publicationDate,
 }: ICategorizableDetailQueryResult<IBookViewModel>): JSX.Element => {
+    const fullScreenHandle = useFullScreenHandle();
+    const [screenState, setScreenState] = useState(false);
```

Comment:
It seems like we would want the full screen functionality to be wrapped in at a higher level. We definitely don't want any of this logic polluting the presenters. I think that the purpose of this PR is more the visual result than the code itself.